packages/manager/apps/key-management-service/src/common/components/drawer/DrawerInnerComponents.component.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { PropsWithChildren } from 'react';
+import { OdsButton } from '@ovhcloud/ods-components/react';
+
+// Sub components for the drawer
+// Those components will be moved to MRC to get a composable version of the drawer
+
+export const DrawerContent = ({ children }: PropsWithChildren) => {
+  return <div className="flex flex-1 flex-col">{children}</div>;
```

Comment:
It's possible add more accessible balise ? As section maybe ?

Reply:
As pointed up in the comment this DrawerContent is a patch for the drawer that will be moved in MRC with a clean implementation.
The idea is to make the drawer more composable.

The Drawer already adds sections for header, content and footer.
If I put a section here it will nest sections, which is not a good practice.

The use of section tags will be clean with the composable version of the drawer.